playbook-website/app/views/pages/visual_guidelines_partials/_opacity.html.erb

```diff
@@ -0,0 +1,75 @@
+   By default, all elements have an opacity value of 1. From 0 to 1, the opacity property controls the transparency level of the element."
+   }) %>
+<br>
+<h4 aria-label="online" class="pb_title_kit_4 pt_sm">Opacity Tokens</h4>
+<br>
+<%= pb_rails("body", props: {
+   text: "Make your own styles using Playbook tokens to keep your site consistent. To use the opacity for your custom css import into your scss file:"
+   }) %><br>
+<%= pb_rails("card", props: {shadow: "deep", margin_bottom: "xl", padding: "none", padding_top: "md" }) do %>
+<ul class="opacity-removing-utilities pb--utlities-color">
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_1"></div>
+      <p class="name">Opacity_1</p>
+      <p class="var">$opacity_1</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_2"></div>
+      <p class="name">Opacity_2</p>
+      <p class="var">$opacity_2</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_3"></div>
+      <p class="name">Opacity_3</p>
+      <p class="var">$opacity_3</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_4"></div>
+      <p class="name">Opacity_4</p>
+      <p class="var">$opacity_4</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_5"></div>
+      <p class="name">Opacity_5</p>
+      <p class="var">$opacity_5</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_6"></div>
+      <p class="name">Opacity_6</p>
+      <p class="var">$opacity_6</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_7"></div>
+      <p class="name">Opacity_7</p>
+      <p class="var">$opacity_7</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_8"></div>
+      <p class="name">Opacity_8</p>
+      <p class="var">$opacity_8</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_9"></div>
+      <p class="name">Opacity_9</p>
+      <p class="var">$opacity_9</p>
+   </li>
+   <li data-name="Data 1" data-variable="$data_1">
+      <div class="pb--color_data_1 opacity_10"></div>
+      <p class="name">Opacity_10</p>
+      <p class="var">$opacity_10</p>
+   </li>
+</ul>
+<% react_contents = Rails.root.join("app/views/pages/code_snippets/user_jsx.txt").read %>
+<%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper", padding_left: "lg" }) do %>
+<div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+   <pre class="highlight">Example in SCSS<p><code>@import "tokens/opacity";</code></p></pre>
+</div>
```

Comment:
Check out how the other files are doing this code sample and replicate. Move the react contents variable to the pages controller as well. Use an actual code sample with an opacity token in action. See the others for example